### DISCUSSION TITLE
Use with NMEA data ?

### DISCUSSION THREAD

**Discussion by theeko74** [nohal/dashboardsk_pi#23] [Q&A] [answered]
Hi Pavel, how to use this plugin with NMEA standard data ?
Like if I want to display COG coming from the GPS ?

**Comment by nohal** [comment, marked as answer]
Translate the data to SignalK. I do not ever plan to implement direct support for any NMEA standard in this code.

**Reply by theeko74** [reply]
Can I translate the NMEA data to SignalK in OCPN?

**Reply by nohal** [reply]
Not in the current core application, externally yes.

**Reply by theeko74** [reply]
Not ideal to have a signalK server running just for that...

**Reply by nohal** [reply]
I think I do not care much what you think about it at this point, this is unreleased code not targeting any currently available environment.

**Reply by theeko74** [reply]
No need to take it that way: I am looking for a way to improve old dashboard layout style without too much effort and wondering if your project could be a good start.
